docs/pages/versions/unversioned/sdk/tracking-transparency.mdx

```diff
@@ -3,7 +3,7 @@ title: TrackingTransparency
 description: A library for requesting permission to track the users on devices using iOS 14 and higher.
 sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-tracking-transparency'
 packageName: 'expo-tracking-transparency'
-platforms: ['android', 'ios']
+platforms: ['android', 'ios', 'tvos']
```

Comment:
If we plan to cherry pick this change into SDK 52, let's also update the versioned doc page.

Reply:
Yeah, I think we can backport this to SDK 52, even though it's a new feature in theory (we usually backport only fixes). The versioned doc page is here: https://github.com/expo/expo/blob/main/docs/pages/versions/v52.0.0/sdk/tracking-transparency.mdx

Reply:
Done